src/Presentation/Detection/Fetching/Resolver.cs

```diff
@@ -11,7 +13,11 @@ public class Resolver : IResolver
 
         public Resolver(IGetDetection getDetection) => _getDetection = getDetection;
 
-        public IEnumerable<MDetection> Execute(IResolveFieldContext<object> input) =>
-            _getDetection.Execute(new Nothing());
+        public IEnumerable<MDetection> Execute(IResolveFieldContext<object> input)
+        {
+            var detections = _getDetection.Execute(new PDetection());
```

Comment:
Should we remove this variable and return the command here ?

Reply:
done